policy/modules/kernel/domain.te

```diff
@@ -259,7 +259,7 @@ optional_policy(`
 
 # allow special io_uring features
 allow unconfined_domain_type domain:io_uring override_creds;
-allow unconfined_domain_type self:io_uring sqpoll;
+allow unconfined_domain_type self:io_uring { cmd sqpoll };
```

Comment:
So, based on my understanding the target context here can be any socket context, which is usually from the `domain` attribute. So in order to cover potential sockets inherited from other domains, we should allow `allow unconfined_domain_type domain:io_uring { cmd }` instead, i.e. add the `cmd` permission to the rule at line 261 instead of 262.